src/assets/i18n/de.json5

```diff
@@ -3838,17 +3838,17 @@
   // "profile.security.form.error.matching-passwords": "The passwords do not match.",
   "profile.security.form.error.matching-passwords": "Die Passwörter stimmen nicht überein.",
   
-  // "profile.security.form.error.password-length": "The password should be at least 6 characters long.",
-  "profile.security.form.error.password-length": "Das Passwort sollte mindestens 6 Zeichen lang sein.",
-  
-  // "profile.security.form.info": "Optionally, you can enter a new password in the box below, and confirm it by typing it again into the second box. It should be at least six characters long.",
-  "profile.security.form.info": "Optional können Sie ein neues Passwort in das Feld darunter eingeben und es durch erneute Eingabe in das zweite Feld bestätigen. Es sollte mindestens sechs Zeichen lang sein.",
+  // "profile.security.form.info": "Optionally, you can enter a new password in the box below, and confirm it by typing it again into the second box.",
+  "profile.security.form.info": "Im nachfolgenden Feld können Sie bei Bedarf ein neues Passwort eingeben und es durch erneute Eingabe in das zweite Feld bestätigen.",
   
   // "profile.security.form.label.password": "Password",
   "profile.security.form.label.password": "Passwort",
   
   // "profile.security.form.label.passwordrepeat": "Retype to confirm",
-  "profile.security.form.label.passwordrepeat": "Zum Bestätigen erneut eingeben",
+  "profile.security.form.label.passwordrepeat": "Zum Bestätigen das Passwort erneut eingeben",
+  
+  // "profile.security.form.label.current-password": "Current password"
+  "profile.security.form.label.current-password": "Ihr aktuelles Passwort"
```

Comment:
@saschaszott : You are missing a comma after this line, which is causing test failures.  This file isn't valid JSON right now.

Reply:
Sorry for that and thank you for reviewing my changes!

Reply:
I've added the missing comma.